helix-term/src/ui/editor.rs

```diff
@@ -736,9 +736,10 @@ impl EditorView {
         // avoid lots of small allocations by reusing a text buffer for each line
         let mut text = String::with_capacity(8);
 
-        for (constructor, width) in view.gutters() {
-            let gutter = constructor(editor, doc, view, theme, is_focused, *width);
-            text.reserve(*width); // ensure there's enough space for the gutter
+        for gutter_type in view.gutters() {
+            let gutter = gutter_type.row_styler(editor, doc, view, theme, is_focused);
```

Comment:
I don't think this is a general rule in the codebase but for me "row_styler" comes across as https://wiki.c2.com/?DontCreateVerbClasses

I prefer the old name of "constructor" but maybe there's a better word for it: "style" or something similar

Reply:
Thanks for the tip! I agree, this name never felt quite right to me. 

When entering into the code base, I found "constructor" to be a bit of a misnomer. I went with the suggested "style", which I quite like.